Vehicle vehicle 
string command
---
bool success
string message